### ISSUE TITLE
Recent Activity - Custom Assets -  line of asset number

### ISSUE THREAD

**Issue by jayavman** [grokability/snipe-it#13824]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Recent Activity on - Custom Asset - Mentioning Line of Input of Custom Asset type. 

### Reproduction steps

1. Recent activity 
2.
<img width="146" alt="Screenshot 2023-11-01 at 5 31 15 pm" src="https://github.com/snipe/snipe-it/assets/114973646/b4cb8708-ef10-4990-b2a6-8ef940ee52b8">

### Expected behavior

The line of the custom asset is not required :
Eg. Asset Condition 10 : should just be Asset condition : 

I think it's actually getting the data from the line of the Custom Asset, It's not even using the line from the Field sets in setting the model restrictions. It's just getting the data from the original custom asset name

### Snipe-IT Version

 v6.2.3 build 11759 (g8c4bf74f9) 

### Operating System

ubuntu

### Web Server

Apache

### PHP Version

8.1.24

**Comment by snipe** [comment]
Yeah, we know - it’s something that’s been on the internal list for a long time, but it’s kind of a pain to handle.
